src/styles/Hero.module.css

```diff
@@ -0,0 +1,67 @@
+/* Keep these for now 
+--clr-neon:hsl(317 100% 54%);
+--clr-bg: hsl(323 21% 16%); 
+*/
+
+.hero {
+  display: table;
+  margin: 0;
+  width: 100%;
+  height: 542px;
+  background-image: url('../../public/images/hero_background.svg');
+  background-size: cover;
+}
+
+@media only screen and (max-width: 600px) {
+  .hero {
+    height: 100%;
+  }
+}
+
+.caption {
+  width: 80%;
+  padding-left: 40px;
+}
+
+.title {
+  color: #fafafa;
+  font-family: var(--font-roboto);
+  font-size: 48px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: normal;
+}
+
+.text {
+  color: #fafafa;
+  font-family: var(--font-roboto);
+  font-size: 24px;
+  font-style: normal;
+  font-weight: 300;
+  line-height: normal;
+}
+
+.col {
+  display: table-cell;
+  padding: 50px 0px 0px 40px;
+}
+
+@media only screen and (max-width: 600px) {
+  .col {
+    display: block;
+    width: 100%;
+  }
+}
+
+.buttonNeonPink {
```

Comment:
if is not finsihed, remove code related with buttons.

Reply:
removed